src/ProjectTemplates/test/Infrastructure/Directory.Build.targets.in

```diff
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="${TemplateTestsPropsPath}" />
```

Comment:
This seems strange. We're importing a `.props` file into a `.targets` project. Should this line be in `Directory.Build.props.in` instead?

I know it might work either way but it could be confusing for props-like things to be imported at an unexpected time in the build process.

Reply:
It needs to be here so that the settings on the project don't override them. This is meant to override the project settings with all the stuff from our build.